app/models/subdomain.rb

```diff
@@ -217,4 +217,9 @@ def self.bootstrap_via_comfy(name, hostname)
   def create_tenant
     Apartment::Tenant.create(self.name)
   end
+
+  def self.add_to_subdomains(subdomain_name)
```

Comment:
@Ayon95 can you rename this to `self.all_with_public_schema` and don't take an argument.  
We don't need one because the `public` schema is special in the sense that it is always present in a Postgres DB (unless explicitly dropped)

Reply:
All right, I'll make those changes.